packages/hathor-rpc-handler/eslint.config.js

```diff
@@ -0,0 +1,14 @@
+// @ts-check
+
+const eslint = require('@eslint/js');
+const tseslint = require('typescript-eslint');
+
+module.exports = tseslint.config(
+  eslint.configs.recommended,
+  ...tseslint.configs.recommended,
+  {
+    ignores: [
+      'eslint.config.js',
```

Comment:
Why do you need to ignore this?

Reply:
Fixed in 61983cabfb8c4b125baca60a8827dcd54ad4e41a